README.rst

```diff
@@ -171,3 +171,21 @@ In order to register the plugin for a ``setup.py``-based project::
 In order to use the parser via command line,
 the ``--url_from-tag`` flag should be used.
 The provided string should be be formatted ``<library>:<method_name>``
+
+Formatting Options
+------------------
+
+If you connect a ``.css`` file to your generated documentation,
+you can make use of certain classes to format parts of the Gherkin
+however you so choose!
```

Comment:
The way I understand, if you put anything in `_static` in your directory that you pass into sphinx, it will copy that over and use it. So we have an arbitrarily named `.css` file in that `_static` directory that makes this work. I can add the bit about `_static` from what I'm seeing, that name is pretty much locked into sphinx.

Reply:
I'm ready to approve pending adding this note about `_static` in.